ci/scenarios/edpm.yml

```diff
@@ -88,7 +75,6 @@ cifmw_edpm_prepare_kustomizations:
             rabbitmq:
               templates:
                 rabbitmq-notifications:
-                  delayStartSeconds: 30
```

Comment:
I'm curious about this field, why are we removing it now. Was it no needed? I think we should add a comment in the commit message with the reason for future reference